apps/src/lib/ui/HelpTip.story.jsx

```diff
@@ -1,98 +1,106 @@
 import React from 'react';
 import HelpTip from './HelpTip';
 
-export default storybook =>
```

Comment:
the diff on this file looks confusing, but it's almost all formatting -- nothing about the story changed